io-engine/src/grpc/v1/stats.rs

```diff
@@ -193,3 +267,23 @@ impl From<ExternalType<(String, BlockDeviceIoStats)>> for IoStats {
         }
     }
 }
+
+/// Returns IoStats for a given BlockDevice.
+async fn get_pool_stats(
```

Comment:
```suggestion
async fn pool_stats(
```

Reply:
This one takes UntypedBdev. So not pool stats method. renamed it to `get_stats`. Lets see if we can use it for replica as well.